app/models/tag.rb

```diff
@@ -407,6 +407,10 @@ def self.graph_data(limit = 250)
     end
   end
 
+  def self.all_tags_by_popularity
+    not_power_tags = Tag.all.order('count DESC').select { |tag| !(tag.name.include? ":") }.uniq(&:name).map(&:name)
```

Comment:
Could you remove the assignment to `not_power_tags` and just keep the expression? Thanks!